scico/linop/_linop.py

```diff
@@ -107,8 +107,10 @@ def valid_adjoint(
 
     by computing :math:`\mathbf{u} = A \mathbf{x}` and
     :math:`\mathbf{v} = A^T \mathbf{y}` for random :math:`\mathbf{x}`
-    and :math:`\mathbf{y}` and confirming that :math:`\| \mathbf{y}^T
-    \mathbf{u} - \mathbf{v}^T \mathbf{x} \|_2 < \epsilon` since
+    and :math:`\mathbf{y}` and confirming that :math:`\frac{\| \mathbf{y}^T
+    \mathbf{u} - \mathbf{v}^T \mathbf{x} \|_2}{\max \left\{
+    \| \mathbf{y}^T \mathbf{u} \|_2, \| \mathbf{v}^T \mathbf{x} \|_2
+    \right\}} < \epsilon` since
```

Comment:
The docstring looks a bit strange like this. I suggest displaying the equation containing the ``\frac{}{}``.

Reply:
Not really sure what you mean. Can you elaborate, please?

Reply:
I meant that it would be better to write using the ``.. math::`` form than the inline ``:math:`` form.

Reply:
I made that and a couple changes in 76337c6.

- clearly different font distinction between matrix (A, A^T) and operators (A, AT)
- removed remedial equation after 'since ...'
- display-equation for the error inequality